providers/aws/components/topic/setup.ftl

```diff
@@ -112,6 +114,37 @@
                                 /]
                             [/#if]
                             [#break]
+
+                        [#case LAMBDA_FUNCTION_COMPONENT_TYPE ]
+                            [#local endpoint = linkTargetAttributes["ARN"] ]
+                            [#local protocol = "lambda"]
+                            [#break]
+
+                        [#case SQS_COMPONENT_TYPE]
+
+                            [#local resourceId = linkTargetResources["queue"].Id ]
+                            [#local policyId =
+                                    formatDependentPolicyId(
+                                        subscriptionId,
+                                        resourceId) ]
+
+                            [#local subscriptionDependencies += [policyId] ]
+
+                            [@createSQSPolicy
+                                id=policyId
+                                queues=resourceId
+                                statements=sqsWritePermission(
```

Comment:
I think the changes to the parameters on the sqsWritePermission don't match this call

Reply:
What did you think was missing? Seems to have the three parameters now required in the update

Reply:
Thought sqswrite permission had four params not three.